contacts/contact_custom_report.sql

```diff
@@ -536,8 +536,7 @@ FROM student_session_aff AS enrollments
   LEFT JOIN emergency_contact_2 ON emergency_contact_2.student_id = enrollments.student_id
 
 WHERE
-  -- get students enrolled in the current academic year
-  sessions.academic_year = 2018
+  enrollments.is_primary_ada = TRUE
```

Comment:
This ensures we filter out duplicate enrollments such as Edward Avitia in WA who had concurrent enrollments.

Reply:
Good catch.